core/Makefile

```diff
@@ -1,7 +1,8 @@
 # exclude submodule sources from *.c wildcard source selection
-SRC := $(filter-out mbox.c msg.c thread_flags.c,$(wildcard *.c))
+SRC := $(filter-out mbox.c msg.c thread_flags.c priority_inheritance.c,$(wildcard *.c))
 
 # enable submodules
 SUBMODULES := 1
+SUBMODULES_NOFORCE := 1
```

Comment:
What is this?

Reply:
it allows for the `SRC` variable to be empty if I remember correctly.